doc/jemalloc.xml.in

```diff
@@ -227,6 +240,20 @@
       allocated memory referenced by <parameter>ptr</parameter> to be made
       available for future allocations.  If <parameter>ptr</parameter> is
       <constant>NULL</constant>, no action occurs.</para>
+
+      <para>The <function>free_sized()</function> function is an extension of
+      <function>free()</function> with a <parameter>size</parameter> parameter
+      to allow the caller to pass in the allocation size as an optimization.
+      Refer to the documentation for <function>sdallocx()</function> for
+      information on the acceptable range of values for
+      <parameter>size</parameter> for a given <parameter>ptr</parameter>.</para>
```

Comment:
I might add a note about the constraints on alignment. In particular, passing a pointer you got from mallocx and friends to this function is maybe safe, maybe unsafe depending on flags. (ditto w.r.t. posix_memalign I suppose, although maybe documenting that bit is posix's job).

Reply:
You're referring to `free_aligned_sized()`, yes?

Reply:
Discussed offline. Documentation has been updated accordingly.